### PR TITLE
VBA Facilities Missing Service Listings

### PR DESCRIPTION
**Pull request by brangi**
## Description
Issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/10416

## Testing done
Locally & http://fbc50130a40c59ea37194446d6ff0d28.review.vetsgov-internal/find-locations/facility/vba_313f 

## Screenshots
Screen Shot 2020-07-21 at 11 03 33 AM

## Acceptance criteria
- [x] When a veteran views a VA benefit facility detail page, services are displayed.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by mmiddaugh** [comment]
Looks good to me!

**Comment by brangi** [comment]
@ncksllvn @ethanteague can you help me review this PR?

File: src/applications/facility-locator/components/ServicesAtFacility.jsx [hunk 1]
```diff
@@ -91,13 +90,28 @@ class ServicesAtFacility extends Component {
       },
     } = this.props;
 
-    if (!services.benefits || isEmpty(services.benefits.standard)) {
+    if (
+      !services.benefits ||
+      services.benefits.length === 0 ||
+      (services.benefits.standard && services.benefits.standard.length === 0)
```
Comment (kelsonic) [inline src/applications/facility-locator/components/ServicesAtFacility.jsx:96]: You can shorten this to:

```js
if (!services?.benefits?.standard) {
  return null;
}
```

BTW really like how you're escaping early here, makes it easy to read 💯

File: src/applications/facility-locator/components/ServicesAtFacility.jsx [hunk 2]
```diff
@@ -91,13 +90,28 @@ class ServicesAtFacility extends Component {
...
+     * supporting v0 as the implementation is still on flipper.
+     *
+     * v0  - services.benefits.standard
+     * v1  - services.benefits
+     */
     return (
       <div className="mb2">
-        <ul>{services.benefits.standard.map(s => this.renderService(s))}</ul>
+        <ul>
+          {(services.benefits.standard &&
```
Comment (kelsonic) [inline src/applications/facility-locator/components/ServicesAtFacility.jsx:111]: Can also write this like: (this is subjective so feel free to disregard if you want lol)

```jsx
<ul>
  {!isEmpty(services?.benefits?.standard) ? (
    services?.benefits?.map(service => this.renderService(service))
  ) : (
    services?.benefits?.standard?.map(service => this.renderService(service))
  )}
</ul>
```

**Review by kelsonic — APPROVED** [review]
LGTM! 🎉

File: src/applications/facility-locator/components/ServicesAtFacility.jsx (hunk 2 above)
Comment (kelsonic) [reply src/applications/facility-locator/components/ServicesAtFacility.jsx:111]: `isEmpty` would be coming from `import { isEmpty } from 'lodash';`
